VK/VK/FriendsList/VC/FriendsListTableViewController.swift

```diff
@@ -61,17 +61,28 @@ final class FriendsListTableViewController: UITableViewController {
     // MARK: - Private Properties
 
     private var sortedFriendsMap: [Character: [UserInfo]] = [:]
+    private var service = NetworkService()
 
     // MARK: - Life Cycle
 
     override func viewDidLoad() {
         super.viewDidLoad()
         makeFriendsSortedMap(friendsInfo: users)
+        fetchData()
     }
 
     // MARK: - Private Methods
-
-//    private func make
+    
+    private func fetchData() {
+        // Friends
+        service.fetchData(method: "friends.get", queryItems: [URLQueryItem(name: "fields", value: "nickname")])
+        // Photos
+        service.fetchData(method: "photos.getAll", queryItems: [URLQueryItem(name: "owner_id", value: "159716695")])
+        // Groups
+        service.fetchData(method: "groups.get", queryItems: [URLQueryItem(name: "owner_id", value: "159716695")])
+        // Groups search
```

Comment:
```suggestion
        service.fetchData(method: "", queryItems: [URLQueryItem(name: "fields", value: "nickname")])
        service.fetchData(method: "", queryItems: [URLQueryItem(name: "owner_id", value: "159716695")])
        service.fetchData(method: "", queryItems: [URLQueryItem(name: "owner_id", value: "159716695")])
```
текстовки в константы

про queryItems контроллер знать не должен, он лишь говорит дай мне друзей, а уже внутри сервиса происходит вся магия с параметрами и т.д.